cmd/neofs-cli/modules/netmap/snapshot.go

```diff
@@ -39,29 +39,35 @@ func prettyPrintNetMap(cmd *cobra.Command, nm netmap.NetMap) {
 
 	nodes := nm.Nodes()
 	for i := range nodes {
-		var strState string
+		PrettyPrintNodeInfo(cmd, &nodes[i], i, "", false)
+	}
+}
+
+func PrettyPrintNodeInfo(cmd *cobra.Command, node *netmap.NodeInfo,
```

Comment:
We would like not to introduce dependencies between `module/<command>` packages, can we move this function in e.g. `internal/common`?